webapp/src/ts/services/target-aggregates.service.ts

```diff
@@ -0,0 +1,324 @@
+import { Injectable } from '@angular/core';
+import * as moment from 'moment';
+import { isString as _isString } from 'lodash-es';
+import { TranslateService } from '@ngx-translate/core';
+
+import { UHCSettingsService } from './uhc-settings.service';
```

Comment:
'@mm-services/....'....
My IDE does this as well when I create a fresh file that I don't import anywhere else in the app (like it doesn't know which project it belongs to and which tsconfig to apply....)

Reply:
I added the alias! Fixed